src/internal/m365/onedrive/item_handler.go

```diff
@@ -133,6 +134,19 @@ func NewRestoreHandler(ac api.Client) *itemRestoreHandler {
 	return &itemRestoreHandler{ac.Drives()}
 }
 
+func (h itemRestoreHandler) PostDrive(
+	context.Context,
+	string, string,
+) (models.Driveable, error) {
+	return nil, clues.New("creating drives in oneDrive is not supported")
```

Comment:
Curious, what's our plan of action if a onedrive drive is deleted between backup & restore?

Reply:
I don't know that it's traditionally possible to delete a onedrive drive.  It's already a very strange case that a user's onedrive can contain more than one... drive...  As a result, I think this handling is appropriate.

That said, this implementation will have to change a bit when we introduce support for moving data between users, since the current handling will error instead of placing the data in the destination user's drive.  That's a change for another PR, however.